app/transformations/ETMPToFrontEnd/API1834Summary.scala

```diff
@@ -29,41 +29,50 @@ object API1834Summary {
   private val readsIsEventTypePresent: Reads[Int] = {
     Reads {
       case JsString(v) => JsSuccess(v.toInt)
+      case JsNumber(v) => JsSuccess(v.toInt)
       case s => JsError(s"Invalid json $s")
     }
   }
-  private def createRow(event: Option[Int], eventType: String): JsObject = {
+  private def createRow(event: Option[Int], eventType: String, numberOfMembers: Option[Int] = None): JsObject = {
     event.fold(Json.obj())(version =>
       Json.obj(
         "eventType" -> eventType,
         "recordVersion" -> version
-      )
+      ) ++ numberOfMembers.map { x => Json.obj("numberOfMembers" -> JsNumber(x))}.getOrElse(Json.obj())
```

Comment:
please provide meaningful variable name for x

Reply:
Changed.